modules/framework/galasa-parent/dev.galasa.framework.api.streams/src/main/java/dev/galasa/framework/api/streams/internal/routes/StreamsTransform.java

```diff
@@ -76,19 +77,19 @@ private StreamData createStreamData(IStream stream) {
 
         StreamData data = new StreamData();
         List<StreamOBRData> streamObrData = new ArrayList<>();
-        String testCatalogUrl = stream.getTestCatalogUrl();
-        String obrLocation = stream.getObrLocation();
+        URL testCatalogUrl = stream.getTestCatalogUrl();
```

Comment:
What if the list of OBRs or the single OBR has a trailing slash ?

Reply:
Addressed in https://github.com/galasa-dev/galasa/pull/236/commits/a9ef197a47c1c7da97f4c1566f80ca24bce91184